report/sections/Preliminaries/preliminaries.tex

```diff
@@ -107,13 +107,15 @@ \section*{Usability Testing}
 
 Where effectiveness regards how well the users accomplished their goal, efficiency regards the resources expended in relation to the completeness of the goals, and lastly, satisfaction regards the comfort of the system to the users.
 To test usability, one may perform a usability test. This is a study of the interaction that users have with a product. It is done to identify usability problems in a system, and results in a list of usability problems and knowledge about how easily a user is able to interact with the system.
-Before the test, the interviewers have to ensure that a test plan, tasks, and questions are ready, and that the equipment works.
+Before the test, the interviewers have to ensure that a test plan is ready.
+They also need to prepare tasks and questions for the test subjects, and ensure that the equipment works.
```

Comment:
What is the equipment? Maybe say necessary equipment is available and functioning? My next question is then what equipment is needed.